.github/workflows/pyopenms-wheels.yml

```diff
@@ -154,6 +154,7 @@ jobs:
         install-deps: 'true'
         modules: 'qtsvg'
         cached: 'false'
+        setup-python: 'false' # fixes an error with newer qt versions
```

Comment:
Is it the qt version or the MacOS version from the GitHub runner that makes problems? Link to issue?